### DISCUSSION TITLE
Milvus client failed to connect to the server using the Kubernetes internal domain URL

### DISCUSSION THREAD

**Discussion by wangchao6856** [milvus-io/milvus#37776] [Q&A and General discussion]
The Milvus client failed to connect to the server using the Kubernetes internal domain URL. However, connecting using the server's POD IP address was successful, and the DNS URL can be successfully pinged from the client's execution environment. The domain name and IP address are correctly mapped. 
Does this mean that the Milvus client does not support accessing via domain names?

**Comment by yhmo** [comment]
Wait @LoveEachDay to comment.

**Reply by LoveEachDay** [reply]
@wangchao6856 Could you specify the detailed error when you are trying to connect to your milvus proxy pod? Add your detailed code snippet here?

**Reply by wangchao6856** [reply]
Temporarily resolved the issue using the ClusterIP and NodePort approach. Thank you.
